docs/guide/developing.md

```diff
@@ -0,0 +1,14 @@
+# Developing with lit-html
+
+## Lit-html
+For developing web component we recommend using the `lit-html` library with the `LitElement` base class.
```

Comment:
"We recommend using the `lit-html` library with the `LitElement` base class for developing web components."